src/content/ja/basic-workflows/sd15-hires-fix.md

```
@@ -0,0 +1,94 @@
+slug: sd15-hires-fix
+navId: sd15-hires-fix
+title: "Hires.fix"
+summary: "Hires.fixを使った高解像度画像生成"
+permalink: "/{{ lang }}/basic-workflows/{{ slug }}/"
+hero:
+  image: ""
+---
+
+## Hires.fixとは？
+
+![](https://gyazo.com/a63d1a6610c9928b6c21ba39a0d533d0){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)